components/google-cloud/storage/upload_dir_contents/component.yaml

```diff
@@ -0,0 +1,32 @@
+name: Upload to GCS
+inputs:
+- {name: Data}
+- {name: GCS path, type: GCSPath}
+- {name: Parallel composite upload threshold, default: '150M'}
+- {name: Delete, type: Boolean, default: False}
```

Comment:
What does this option do? Can we try to make it easier to understand?

Reply:
It ensures that any objects in the remote bucket that aren't present locally are removed. Let's use the example above, but suppose the remote directory already looks like this:

```
my-bucket
  some-folder
    1
    3
    5
```

and we're uploading the contents of:

```
tmp
  1
  2
  3
```

without `Delete` set, we get:

```
my-bucket
  some-folder
    1
    2
    3
    5
```

with it, we'll have:

```
my-bucket
  some-folder
    1
    2
    3
```

Definitely up for clarifying/renaming, any suggestions?